### ISSUE TITLE
A onClose() callback or similar?

### ISSUE THREAD

**Issue by RaXaR** [tameraydin/ngToast#72]
Hi :)  As the title suggests, could you add a callback for when a toaster is closed/removed?
I was thinking of something down the line of:
```
ngToast.create({
    content: 'Some random content',
    timeout: 1200,
    className: 'my-toaster-class',
    onClose: myFunction
});
```

**Comment by tameraydin** [comment]
I agree, I don't think it would bring too much code so it would be a nice & essential feature:) Feel free to create a PR with tests!

**Comment by tameraydin** [comment]
Available with [2.0.0](https://github.com/tameraydin/ngToast/releases/tag/2.0.0)
